packages/logging/src/index.js

```diff
@@ -163,12 +166,7 @@ Logging.prototype.createSink = function(name, config, callback) {
 
   delete reqOpts.sink.gaxOptions;
 
-  this.request({
-    client: 'configServiceV2Client',
-    method: 'createSink',
-    reqOpts: reqOpts,
-    gaxOpts: config.gaxOptions
-  }, function(err, resp) {
+  this.api.Config.createSink(reqOpts, config.gaxOptions, function(err, resp) {
```

Comment:
Interestingly, all `resp`s are coming back as arrays. I'm not sure why, but it's forcing me to unbox each. I think that's either a bug in GAX or the code you wrote, and I possibly butchered @callmehiphop. Please let me know if you have any thoughts.